README.md

```diff
@@ -532,6 +532,36 @@ ctest
 cmake --install . --prefix <path_to_install_dir>
 ```
 
+### Building for CUDA
+
+- On Linux*
+
+With the CUBLAS backend:
```

Comment:
Let's align names 
In this readme above we have cuBLAS (not CUBLAS) and cuRAND (not CURAND)

Reply:
Good point, updated with your suggestion